projects/pazznetwork/ngx-chat/src/lib/services/adapters/xmpp/plugins/message-state.plugin.ts

```diff
@@ -62,52 +60,56 @@ export class MessageStatePlugin extends AbstractXmppPlugin {
             .subscribe((event) => this.handlePubSubEvent(event));
     }
 
-    async onBeforeOnline(): Promise<any> {
+    async onBeforeOnline(): Promise<void> {
         this.parseContactMessageStates().catch(err => this.logService.error('error parsing contact message states', err));
     }
 
-    private async parseContactMessageStates() {
+    private async parseContactMessageStates(): Promise<void> {
         const itemElement = await this.publishSubscribePlugin.retrieveNodeItems(STORAGE_NGX_CHAT_CONTACT_MESSAGE_STATES);
         this.processPubSub(itemElement);
     }
 
-    private processPubSub(itemElement: Element[]) {
-        const results: JidToMessageStateDate = Object.create(null);
+    private processPubSub(itemElement: Stanza[]): void {
```

Comment:
Rename parameter `itemElement` to `itemElements`.

Reply:
Done